Backend/app/therapy_sessions/views.py

```
@@ -1,3 +1,1541 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
```

Ruff (0.12.2)

`django.shortcuts.render` imported but unused

Remove unused import: `django.shortcuts.render`

(F401)